docs/console-api/openapi/experiments/experiments-data.json

```diff
@@ -2678,9 +2676,9 @@
                   }
                 },
                 "properties": {
-                  "groupID": {
+                  "id": {
                     "type": "string",
-                    "description": "The groupID to be selected as the winner"
+                    "description": "The id of group to be selected as the winner"
```

Comment:
keep this as before? Seemed fine and new text has a grammar error. Or maybe just insert a space so "The groupID to be selected as the winner"